source/fundamentals/crud/write-operations/insert.txt

```diff
@@ -4,9 +4,194 @@ Insert a Document
 
 .. default-domain:: mongodb
 
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+   
 Overview
 --------
 
-In this section, we show you how to call the write operations to **insert**
-documents from a collection in your MongoDB database.
+In this guide, you can learn how to **insert** documents into a MongoDB
+collection.
 
+Before you can retrieve, update, or delete information such as user
+profiles or orders in MongoDB, you first need to perform an insert
+operation to load data. An insert operation inserts one or more
+documents into a MongoDB collection using one of the following methods:
+``InsertOne()``, ``InsertMany()``, or ``BulkWrite()``.
+
+The following sections focus on ``InsertOne()`` and ``InsertMany()``. For an
+example on how to use the ``BulkWrite()`` method, see the :doc:`Bulk
+Operations Usage Example </usage-examples/bulkWrite>`.
+
+A Note About ``_id``
+~~~~~~~~~~~~~~~~~~~~
+
+When inserting a document, MongoDB enforces one constraint on your
+documents by default. Each document *must* contain a unique ``_id``
+field.
+
+There are two ways to manage this field:
+
+- You can manage this field yourself, ensuring each value you use is unique.
+- You can let the driver automatically generate unique ``ObjectId`` values.
+
+Unless you provide strong guarantees for uniqueness, we recommend
+you let the driver automatically generate ``_id`` values.
+
+.. note::
+
+   Duplicate ``_id`` values violate unique index constraints, which
+   causes the driver to throw a ``WriteError``.
+ 
+For more information about the ``_id`` field, see the Server Manual Entry on
+:manual:`Unique Indexes </core/index-unique/>`.
+
+Insert a Single Document
+------------------------
+
+Use the ``InsertOne()`` method to insert a single document into a collection.
+
+Upon successful insertion, the method returns an
+``InsertOneResult`` instance that contains the ``_id`` of
```

Comment:
Suggestion: Similar to what you did for the change a document page, I think you should list out the properties and whey are here

Applies to the InsertMany section too